includes/Admin/Admin_Page.php

```diff
@@ -272,6 +272,26 @@ public function admin_footer() {
 				'type' => 'text/template',
 			)
 		);
+
+		// Print check's page styles.
+		echo '<style>';
+		echo '
```

Comment:
Nit-pick, but better to directly output rather than using `echo`. For instance, it gives you syntax highlighting in most IDEs.
```suggestion
		// Print admin page styles.
		?>
		<style>
```